df.md

```diff
@@ -0,0 +1,49 @@
+# 🦈 Senior Product Developer - Data Farming 
+[Ocean Protocol](https://www.oceanprotocol.com) builds tools for people to reclaim control of their data and unleash an open data economy. Ocean tools make it easy to publish & consume data services as data access tokens, and bridge data to DeFi. 
+
+Ocean has a vibrant, expanding ecosystem of data suppliers, data consumers, stakers, and community developers. To serve it better, we’re hiring! If you are someone that loves to build things and cares deeply about the future of data, we want to hear from you.
+
+
+## Responsibilities
+* Candidate will have to communicate with different teams to sync on roadmap delivery
+* Candidate will need to engage with the community to get faster feedback to understand usage patterns, pain points, and user experience
+* They will need to understand to differentiate good and bad feedback and find real pain points and needed improvements
+* They will improve and evolve product/program based on learnings from usage data and user feedback 
+* They will take ownership of related tech like scripts and manage and update scripts and reward functions to implement design changes and effectively communicate to the community about changes
+* They will have to execute scripts to automate some web3 tasks like token distribution
+* They will have to identify fraudulent actors taking unfair advantage of the program and make improvements to curb them
+* Over time they will ideally automate/delegate parts or sections of the program for optimization. Some things can be automated using script and tech. They will design initiatives to decentralize other decision-making efforts to the community via panel or voting.
```

Comment:
The list of responsibilities is too long. Yet still weirdly vague.

Here's a new set, inspired by https://github.com/oceanprotocol/jobs/blob/main/sr-dev.md:
- You will design & build software for Ocean Protocol Data Farming, including scripts for reading on-chain data, performing analysis, token distribution, and whatever else is needed for the Data Farming process. The scope of the software will grow over time, based on feedback from the community, e.g. to do on-chain voting and interact with more protocols.
- You will co-plan and co-design the product with the Data Farming team in an agile, open-source development process